### PR TITLE
added popToViewController:usingTransition: method

### PR DESCRIPTION
**Pull request by itchingpixels**
Hi,

I have been using STPTransitions and it's awesome - but I needed to implement circular navigation in a project and ended up needing a popToViewController: usingTransition: method, so I added it to the UINavigationController category.
Here it is:)

### CODE REVIEW

File: Core/UINavigationController+STPTransitions.h [hunk 1]
```diff
@@ -6,6 +6,10 @@
 
 - (void)pushViewController:(UIViewController *)viewController
            usingTransition:(STPTransition *)transition;
+
+- (void)popToViewController:(UIViewController*)viewController
+		   usingTransition:(STPTransition *)transition;
```
Comment (stepanhruda) [inline Core/UINavigationController+STPTransitions.h]: Could you align the colons here?

**Comment by itchingpixels** [comment]
sure! :)

File: Core/UINavigationController+STPTransitions.h [hunk 2]
```diff
@@ -6,6 +6,10 @@
 
 - (void)pushViewController:(UIViewController *)viewController
            usingTransition:(STPTransition *)transition;
+
+- (void)popToViewController:(UIViewController*)viewController
+           usingTransition:(STPTransition *)transition;
```
Comment (stepanhruda) [inline Core/UINavigationController+STPTransitions.h]: Haha, I don't mean to be _that guy_, but they're supposed to be underneath each other. Xcode does this automatically when you press the tab key.

**Comment by stepanhruda** [comment]
Never mind the spacing, this is useful, thanks!

**Comment by itchingpixels** [comment]
sure, sorry about not responding sooner, I just got reminded this is outstanding by your response, I had a very busy few days:/
feel free to adjust it, the reason why it didn't align properly because I was doing it in Github's code editor (probably a mistake)

or I can do it now, if it makes sense.

**Pull request merged**